ICSharpCode.CodeConverter/CSharp/ExpressionNodeVisitor.cs

```diff
@@ -651,7 +651,7 @@ public override async Task<CSharpSyntaxNode> VisitInvocationExpression(VBasic.Sy
                 return await CreateElementAccess();
             }
 
-            if (expressionSymbol != null && expressionSymbol.IsKind(SymbolKind.Property)) {
+            if (expressionSymbol != null && expressionSymbol.IsKind(SymbolKind.Property) && invocationSymbol.GetParameters().Length == 0) {
```

Comment:
I have a nagging feeling that there's a more general case that may still be missed. I wonder whether this if statement should be below the other one. I'll merge this since it's definitely better than it was, just wondering if you have any thoughts on more general cases - I'll post here if I think of a specific case where it's still broken.

Reply:
I can't think of anything off the top of my head. I think you're probably right about the ordering - though unless we think of another case I don't think it currently makes a difference.

Reply:
Yep, I've thought about it a bit more and they seem mutually exclusive so I think this covers it, thanks!